<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSingleParamDialog</class>
 <widget class="QDialog" name="SelectSingleParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>737</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_paramNameLabel">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>395</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>参数列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_parameditor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>9</y>
     <width>461</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_AllSystemRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>所有参数表</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="m_C3IRadioButton">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>42</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>C3I系统[1-10000]</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="m_DFZXRadioButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>42</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>东风中心[10001-20000]</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="m_KELRadioButton">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>43</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>库尔勒[20001-30000]</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="m_CustomRadioButton">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>43</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义参数表范围</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="m_CustomMinParamEdit">
   <property name="geometry">
    <rect>
     <x>771</x>
     <y>40</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_CustomMaxParamEdit">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>40</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_SelectParamButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LineLable">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>44</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_formTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>941</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_paramTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>414</y>
     <width>941</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_ParamLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>14</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>参数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectSingleParamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectSingleParamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
